emily/handler/src/api/handlers/deposit.rs

```diff
@@ -242,6 +252,42 @@ pub async fn create_deposit(
         .map_or_else(Reply::into_response, Reply::into_response)
 }
 
+/// Parameters from the deposit and reclaim scripts.
+struct ScriptParameters {
+    max_fee: u64,
+    recipient: String,
+    lock_time: i64,
+}
+
+/// Convert scripts to resource parameters.
+///
+/// This function is used to convert the deposit and reclaim scripts into the
+/// parameters that are stored in the database.
+fn scripts_to_resource_parameters(
+    deposit_script: &String,
+    reclaim_script: &String,
```

Comment:
Typically, you would use a `&str` here instead of a `&String`. I'm actually surprised that clippy didn't through a warning here.

Reply:
Done!